meetings/NPM_websockets.md

```diff
@@ -0,0 +1,27 @@
+# Paquetes NPM + Websockets y javascript
+  - [Link](http://www.meetup.com/Beer-JS-Bogota/)
+  - Fecha: 10 Marzo 2016
+  - Lugar: [Café los perritos]()
+
+## Propuesta
+Nuestro segundo encuentro del año tratará dos temas de alto interés:
+
+### Introducción a los paquetes NPM
+_Miguel Asencio_
+
+Esta charla trata de como funciona un paquete en Node.js, como llenar un archivo `package.json`, como escribir un script NPM y como subirlo a NPM.
+
+### Websockets y javascript:
+_Ignacio Galieri_
+
+TODO
```

Comment:
Esta charla se trata sobre la utilización de WebSockets para lograr un integración entre BE (backend) y FE (frontend).